open-craft/sprints
# [BB-1719] Dashboard follow ups

This implements the following improvements:
- back buttons for sprint and user's board,
- bot directive for remaining review time (working when the ticket is in `External Review` or `Merged`,
- partial vacations - if the user's day starts before the sprint meeting, the proportional amount of hours is counted for the previous/next sprint respectively.

## Testing instructions:
1. Set `.env` file (you can find it in a comment on the ticket).
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Set frontend/.env.local file (you can find it in a comment on the ticket).
1. _Optional (this will speed up testing, as the accounts' rework is not merged into the master yet)_: remove lines 82 and 83 from `frontent/src/App.js`.
1. Navigate to frontend and run `npm i && npm start`.
1. Log in.
1. Go to the cell's and user's dashboard and check that `Go Back` button works correctly.
1. Check for Josh that BB-1591 has remaining review time (see the beginning of that ticket description).
1. Compare vacations for Jill (vacation time for the next sprint is reduced by today, as this is a part of the current sprint).
